### PR TITLE
Added toggleable guidance/comments section

### PR DESCRIPTION
**Pull request by benjaminfaure**
Changes proposed in this PR:

- Add toggleable comments/guidance section
  - add config to enable/disable this feature
  - add config to open/close by default
  - question/guidance divs use flexbox instead of bootstrap grid system

### CODE REVIEW

**Comment by benjaminfaure** [comment]
I wasn't sure where to put the CSS & JS code, if it's in the wrong place please tell me :)

File: config/initializers/_dmproadmap.rb [hunk 1]
```diff
@@ -122,6 +122,10 @@ class Application < Rails::Application
     config.x.application.require_contributor_name = false
     config.x.application.require_contributor_email = false
 
+    # Defines if Guidances/Comments in toggleable & if it's opened by default
+    config.x.application.guidance_comments_toggleable = true
+    config.x.application.guidance_comments_opened = true
```
Comment (briri) [inline config/initializers/_dmproadmap.rb:127]: Can you change the name of this variable to `guidance_comments_opened_by_default`?

**Review by briri — CHANGES_REQUESTED** [review]
Thanks for getting this to us so quickly @benjaminfaure!

I have 2 very minor changes. It looks great overall and the JS and SCSS files are in the correct place! :)

I will pull down the changes and confirm that it works as expected in the UI later today

**Comment by benjaminfaure** [comment]
Thanks for your feedback @briri , I push the changes you requested :)

**Comment by raycarrick-ed** [comment]
@benjaminfaure @briri Pulled down the code on top of an up to date roadmap and it worked perfectly out of the box. Very nice! So I'm for it to get merged in.

**Review by briri — APPROVED** [review]

**Pull request merged**